packages/vite/src/node/plugins/html.ts

```diff
@@ -671,3 +690,7 @@ function serializeAttrs(attrs: HtmlTagDescriptor['attrs']): string {
   }
   return res
 }
+
+function incrementIndent(indent: string = '') {
+  return `${indent}${indent[0] === '\t' ? '\t' : '  '}`
```

Comment:
Is it important to support also indentations like 3 or 4 spaces?

Reply:
I dont know if it is worth to add even more complexity, maybe we should later read a editorconfig file and use that instead of adding an option?

Reply:
Okay, let's keep this in mind, or maybe we could add a TODO with a link to here 🤔